roles/repository/tasks/main.yml

```diff
@@ -371,22 +396,6 @@
         owner: "{{ username }}"
         group: "{{ group_name }}"
 
-    - name: >-
-        Insert or Update custom properties added through
-        configuration_files/alfresco-global.properties
-      ansible.builtin.blockinfile:
-        path: "{{ settings_folder }}/classpath/alfresco-global.properties"
-        marker: "{mark} - ANSIBLE MANAGED BLOCK"
-        marker_begin: "### Begin - Custom user properties"
-        marker_end: "### End - Custom user properties"
-        insertafter: EOF
-        block: >-
-          {{ lookup('file', role_path + '/../../configuration_files/alfresco-global.properties') }}
```

Comment:
update https://github.com/Alfresco/alfresco-ansible-deployment/blob/master/docs/playbook-upgrade.md that we are getting rid of this file and the properties should be moved to new structure